src/main/java/appeng/client/gui/implementations/GuiInterfaceTerminal.java

```diff
@@ -29,8 +29,14 @@
 import java.util.Set;
 import java.util.WeakHashMap;
 
+import appeng.container.slot.AppEngSlot;
+import appeng.container.slot.SlotCraftingMatrix;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'appeng.container.slot.SlotCraftingMatrix'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AUselessImportCheck)